spec/javascripts/reducers/column_backlog_spec.js

```diff
@@ -63,14 +81,49 @@ describe('Backlog Column reducer', () => {
         startDate: "2018-09-03T16:00:00",
         iterationLength: 1,
         defaultVelocity: 2,
-      },
+      }
     }
-  }
+  };
+
+  function toggleStoryAction(id) {
+    return {
+      type: actionTypes.TOGGLE_STORY,
+      id
+    }
+  };
+
+  describe("Toggle a story", () => {
+    it("expand", () => {
```

Comment:
We could add some contexts here, like `when story is collapsed` and `when story is expanded` around the `it` to better readability 😄